ui/pages/unlock-page/unlock-page.util.js

```diff
@@ -0,0 +1,17 @@
+export function getCaretCoordinates(element, position) {
+  const div = document.createElement('div');
+  div.id = 'password-mirror-div';
+  document.body.appendChild(div);
+  const computed = window.getComputedStyle(element);
+  div.textContent = new Array(position + 1).join('•');
+  const span = document.createElement('span');
+  span.textContent = '•';
+  div.appendChild(span);
+
+  const coordinates = {
+    top: span.offsetTop + parseInt(computed.borderTopWidth, 10),
+    left: span.offsetLeft + parseInt(computed.borderLeftWidth, 10),
+  };
+  document.body.removeChild(div);
```

Comment:
i would do `div.remove()` instead so the removal operation will be independent of who's the parent

Reply:
DIV

Reply:
i would do div.remove() instead so the removal operation will be independent of who's the parent